src/components/FavoriteArticles/FavoriteArticles.js

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment } from 'react';
+import styles from '../UserProfileTabs/UserProfileTabs.module.scss';
+import apple from '../../assets/images/apple.jpg';
```

Comment:
Please add comments between these imports. e.g
```
import React, { Fragment } from 'react';

// styles
import styles from '../UserProfileTabs/UserProfileTabs.module.scss';

// images
import apple from '../../assets/images/apple.jpg';
```

Reply:
Okay. I will implement this fix and also do this moving forward